webpack/assets/javascripts/react_app/components/PasswordStrength/__tests__/__snapshots__/PasswordStrength.test.js.snap

```diff
@@ -17,6 +17,49 @@ exports[`PasswordStrength component rendering renders password-strength 1`] = `
       defaultValue=""
       inputProps={
         Object {
+          "autoComplete": undefined,
```

Comment:
can you update the test so it won't be `undefined` ?

Reply:
`"autoComplete": undefined` updated to `"autoComplete": ""`

> I wonder if we should always disable auto-completion for new-passwords

This change is only at one place for user edit form, it won't affect any other forms anywhere else.

Reply:
so isn't the same issue present on other password fields rendered using this component? if so, it may be good to fix it everywhere as @laviro suggests

Reply:
my point is that the `PasswordStrength` component is used only when a user creates a new password. 

It sounds like a bad practice to store that password in the browser so I believe we should disable it the way you did to all password strength,
and maybe even open a PR to `react-password-strength` with this suggestion

Reply:
I updated PR, I realized that there is easier solution how to do it, change is now much more simple, just adding `autComplete` to `inputProps`
`inputProps={{ name, id, className, autoComplete: 'new-password' }}`